Source/OCMock/OCMMacroState.m

```diff
@@ -40,21 +41,31 @@ + (OCMStubRecorder *)endStubMacro
     OCMStubRecorder *recorder = [[(OCMStubRecorder *)[globalState recorder] retain] autorelease];
     BOOL didThrow = [globalState invocationDidThrow];
     [threadDictionary removeObjectForKey:OCMGlobalStateKey];
-	if(didThrow == NO && [recorder wasUsed] == NO)
-	{
-		[NSException raise:NSInternalInconsistencyException
-					format:@"Did not record an invocation in OCMStub/OCMExpect/OCMReject.\n"
-						   @"Possible causes are:\n"
-						   @"- The receiver is not a mock object.\n"
-						   @"- The selector conflicts with a selector implemented by OCMStubRecorder/OCMExpectationRecorder."];
-	}
+	  if(didThrow == NO && [recorder wasUsed] == NO)
+	  {
+      OCMLocation *location = recorder.ocm_location;
+        NSString *explanation = @"Did not record an invocation in OCMStub/OCMExpect/OCMReject.\n"
+                                @"Possible causes are:\n"
+                                @"- The receiver is not a mock object.\n"
+                                @"- The selector conflicts with a selector implemented by OCMStubRecorder/OCMExpectationRecorder.";
```

Comment:
Similarly I would prefer that we would assert on beginStubMacro/beginExpectMacro/beginRejectMacro that the selector conflicts

Reply:
Can you give me an outline of how that would work?